assets/dev/js/editor/container/container.js

```diff
@@ -419,6 +470,28 @@ export default class Container extends ArgsObject {
 		return false;
 	}
 
+	/**
+	 * Function forSomeChildrenRecursive().
+	 *
+	 * Will run over children recursively, breaks if the callback return true.
+	 *
+	 * @param {function(container:Container)} callback
+	 *
+	 */
+	forSomeChildrenRecursive( callback ) {
```

Comment:
Maybe it should return `true` or `false` like: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Handled.